src/js/pages/Escrow/index.scss

```diff
@@ -4,4 +4,8 @@
   p.balanceAlert {
     color: #FF2D55;
   }
-}
\ No newline at end of file
+
+  .disabled {
+    opacity: .4;
```

Comment:
Using `.text-muted` should do the same effect

Reply:
Ah! I'm using `.disabled` to to set the image of the button as semitransparent too

Reply:
Ah ok, I didn't notice. It's fine then!